make-nagios-commands-cfg.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+
+smtp_server = os.getenv('SMTP_SERVER', 'localhost')
```

Comment:
Unsure about this tbh. Being able to send email notifications isn't a mandatory part of nagios even though we do want to have it configurable. Alternative to have it fallback here would be to add default setting to the openshift template, or allow this to work if the smtp settings don't exist.